app/controllers/api/api_controller.rb

```diff
@@ -17,6 +17,10 @@ class ApiController < ActionController::Base
       render_error 404, 'Not found'
     end
 
+    rescue_from ActiveRecord::RecordInvalid do |e|
+      render_error 422, e.record.errors.full_messages
```

Comment:
@alan full messages will be an array, where elsewhere message is assumed to be a string. We're not using this programatically at this stage, so what do you think about keeping the types consistent but keeping it simple and simply returning `full_messages.join(', ')`?